sdk/tables/ci.yml

```diff
@@ -38,3 +38,6 @@ stages:
   - template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
     parameters:
       ServiceDirectory: tables
+      Artifacts:
+        - name: azure-tables
+          safeName: azuretables
```

Comment:
We decided to remove the artifacts for Tables for now since we're still debating on the package name, and didn't want to publish a dev build yet.

Reply:
I think it will be better to disable scheduled builds on the pipeline itself and also `SetDevVersion` variable to false. This way you still get PR and CI runs but not internal run that published to the Dev Feed.
@weshaggard  what do you think?

Reply:
@mahdiva Can we set the private field in the package.json for this sdk? This will prevent it from getting published to npm even as a dev package

Reply:
I have included that change in the PR

Reply:
Thanks @chidozieononiwu and @KarishmaGhiya !